includes/wikia/models/DesignSystemGlobalFooterModel.class.php

```diff
@@ -2,6 +2,8 @@
 
 class DesignSystemGlobalFooterModel extends WikiaModel {
 	const DEFAULT_LANG = 'en';
+	const WIKI_PRODUCT = 'wikis';
+	const FANDOM_PRODUCT = 'fandoms';
```

Comment:
Fixed the fandoms inconsistency here: https://github.com/Wikia/chef-repo/pull/11089/files. Thanks for catching @sebastianmarzjan!